src/main/java/Hamyo.java

```diff
@@ -0,0 +1,68 @@
+import Hamyo.Misc.HamyoException;
+import Hamyo.Misc.Parser;
+import Hamyo.Misc.UI;
+import Hamyo.Misc.Storage;
+
+import Hamyo.Tasks.TaskList;
+
+import java.util.Scanner;
+
+/**
+ * Main class of the Hamyo Chatbot.
+ *
+ * @author Han Yu.
+ */
+public class Hamyo {
+
+    private final UI ui;
+    private final Storage storage;
+    private final TaskList tasks;
+    private boolean active;
```

Comment:
I think the name of a boolean variable should be named to sound like a booleans.